app/layout.tsx

```diff
@@ -37,17 +37,13 @@ export default function RootLayout({
   return (
     <html lang="he" dir="rtl" suppressHydrationWarning={true}>
       <body
-        className={`min-h-screen m-0 darkmode-main ${birzia.className} ${inter.variable}`}
+        className={`min-h-screen w-full flex flex-col items-center m-0 darkmode-main ${birzia.className} ${inter.variable}`}
       >
         <ThemeProvider attribute="class" defaultTheme="system" enableSystem>
-          <header>
-            <Header />
-          </header>
-          <main className="h-full pt-24 md:pt-32">{children}</main>
+          <Header />
+          <main className="flex-1 h-full pt-24 md:pt-32">{children}</main>
```

Comment:
flex-1 :+1:
h-full is unneeded since you expect the parent to take care of the height with flex-grow:1 
Which it will thanks to you putting body at min-h-screen :)